src/implementations/twig/components/media-container/media-container.html.twig

```diff
@@ -93,6 +99,10 @@
 
 {# Print the result #}
 
+{% if _expandable is not empty %}
```

Comment:
can we keep the html indentation here? this shouldn't be indented basically, same for the closure tag.
The solution is not that clean, adding a wrapper conditionally as an element for the whole component, but i see that it was somehow needed in order to accomplish this.. 
But what if we add it anyhow, what if we set the root classes (or part of them) to this div instead of the figure for all the variants? Aslo because it has zero styles associated, currently...